.github/workflows/release-note-label.yml

```diff
@@ -0,0 +1,62 @@
+# A workflow to validate only one release-note label is selected
+
+name: Release-note label
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - edited
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Validate
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const { body, user } = context.payload.pull_request;
+
+            // Skip validation on pull requests created by the mlflow-automation bot
+            if (user.login === "mlflow-automation") {
+              console.log("Skipping validation since this pull request was created by the mlflow-automation bot");
+              return;
+            }
+
+            const RNLabels = [
+              "rn/feature",
+              "rn/breaking-change",
+              "rn/bug-fix",
+              "rn/documentation",
+              "rn/none",
+            ];
+
+            // Extract selected release-note labels
+            const regexp = /^- \[(.*?)\] ?`(.+?)`/gm;
+            const selectedRNLabels = [];
+            let match = regexp.exec(body);
+            while (match != null) {
+              const selected = match[1].trim().toLocaleLowerCase() === "x";
+              const name = match[2].trim();
+
+              if (RNLabels.includes(name) && selected) {
+                selectedRNLabels.push(name);
+              }
+              match = regexp.exec(body);
+            }
+
+            console.log("Selected release-note labels:");
+            console.log(selectedRNLabels);
+
+            // If the number of selected release-note labels is not 1, set the action status to "failed"
+            const numSelected = selectedRNLabels.length;
+            if (numSelected === 0) {
+              core.setFailed("No release-note label is selected. Please select one.");
+            } else if (numSelected > 1) {
+              core.setFailed("Multiple release-note labels are selected. Please select one.");
+            }
```

Comment:
This shows an error message like this:

![Screen Shot 2020-12-25 at 9 50 44](https://user-images.githubusercontent.com/17039389/103112155-b6274e00-4696-11eb-95d3-15c1ba611553.png)

Reply:
One might think "what is a release-note label?". One way to address this issue is:

Update the PR template as follows:

```diff
- ### Select release note label
+ <h3 id="release-note-category">Select release note label</h3>
```

This allows us jump to the release-note category section with an URL like `https://.../pull/xxx#release-note-category`.